modules/repofiles/upload.go

```diff
@@ -55,6 +55,23 @@ func UploadRepoFiles(repo *models.Repository, doer *models.User, opts *UploadRep
 		return fmt.Errorf("GetUploadsByUUIDs [uuids: %v]: %v", opts.Files, err)
 	}
 
+	names := make([]string, len(uploads))
+	infos := make([]uploadInfo, len(uploads))
+	for i, upload := range uploads {
+		names[i] = upload.Name
+		infos[i] = uploadInfo{upload: upload}
+
+		// Check file is not lfs locked
+		filepath := path.Join(opts.TreePath, upload.Name)
```

Comment:
Move these before line 61?

Reply:
I presume that this was referring to some other lines which have been moved? (presumably lines 58 & 59)

As `filepath` has to be set within the loop as it uses `upload.Name`